### ISSUE TITLE
Doesn't work with boombox controller mod

### ISSUE THREAD

**Issue by HamedAlmanthari** [EBro912/QuickRestart#2]
hi, can you make this mode compatible with boombox controller mod? i don't know how to contact boombox controller creator in order to ask them if they could make the mod work with this mod otherwise i would ask them if they could. also i'm not sure if that is even possible. it would be much appreciated.

**Comment by EBro912** [comment]
If you're certain that Boombox Controller is the culprit, I can try and get in touch with the developer to sort things out.

**Comment by HamedAlmanthari** [comment]
yes i am almost 100% certain. when i enable boombox controller mod, quick restart does not work, only boombox controller does. when i disable boombox controller, quick restart works. ty

**Comment by EBro912** [comment]
I believe I know the reason behind the incompatibility, I will try and make a workaround and release it as soon as I can.

**Comment by HamedAlmanthari** [comment]
aight ty so much
